song-docs/source/introduction.rst

```diff
@@ -0,0 +1,156 @@
+==============
+Introduction
+==============
+
+
+What is SONG?
+======================
+
+SONG is a robust metadata and validation system used to quickly and reliably track genome metadata scattered across multiple cloud storage systems. 
+In the field of genomics and bioinformatics, metadata managed by simple solutions such as spreadsheets and text files require significant time and effort to maintain and ensure the data is reliable. 
+With several users and thousands of genomic files, tracking the state of metadata and their associations can become a nightmare. 
+The purpose of SONG is to minimize human intervention by imposing rules and structure to user uploads, which as a result produces high quality and reliable metadata with a minimal amount of effort.
+
+.. note::
+
+    **SONG** is a recurrsive acronym for **S**\ ONG's **O**\ ur **N**\ ew **G**\ NOS
+
+.. 
+    What SONG is NOT
+    ==================
+
+.. _introduction_features:
+
+Features
+======================
+
+- Synchronous and asynchronous metadata validation using `JsonSchema <http://json-schema.org>`_
+- Strictly enforced data relationships and fields
+- Optional **info** json fields for user specific metadata
```

Comment:
Why is info in bold ? Maybe link to the corresponding section of the documentation.